ui/app/lib/kv-object.js

```diff
@@ -3,6 +3,7 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
+// TODO kv engine cleanup: Remove in favor of core/utils/kv-object
```

Comment:
I left this instead of deleting it since we're in a sidebranch, and I didn't want to pull this out from under anyone's feet in case it's being used in other work concurrently